packages/nc-gui/components/cell/GeoData.vue

```diff
@@ -1,6 +1,8 @@
 <script lang="ts" setup>
 import type { GeoLocationType } from 'nocodb-sdk'
 import { Modal as AModal, latLongToJoinedString, useVModel } from '#imports'
+import GpsFixedIcon from '~icons/mdi/gps-fixed'
+import OpenInNewIcon from '~icons/mdi/open-in-new'
```

Comment:
You don't need to import for mdi icons if they are used in template. They are imported in components.d.ts automatically when you declare them. Therefore, you may remove L4 and L5.